package.json

```diff
@@ -41,7 +41,7 @@
   "homepage": "https://github.com/mapbox/quilt#readme",
   "dependencies": {
     "@mapbox/sphericalmercator": "^1.1.0",
-    "mapnik": "4.3.1",
+    "mapnik": "^4.5.4",
```

Comment:
Was this updated to capture dropped Node < 10 support introduced in 4.4.0? Any reason not to bump this to 4.5.5?

Reply:
Ah, didn't see this before I merged yesterday. This was updated because of binary compatibility issues with newer versions of Node & Mapnik. We could have bumped to 4.5.5 no problem and still supported Node v12 & v14. That was purely an oversight on my end.